database/sql/src/main/scala/cromwell/database/slick/CallCachingSlickDatabase.scala

```diff
@@ -49,10 +49,12 @@ trait CallCachingSlickDatabase extends CallCachingSqlDatabase {
   }
 
   override def invalidateCall(callCachingEntryId: Int)
-                               (implicit ec: ExecutionContext): Future[Unit] = {
-    import cats.syntax.functor._
-    import cats.instances.future._
-    val action = dataAccess.allowResultReuseForCallCachingEntryId(callCachingEntryId).update(false)
-    runTransaction(action) void
+                               (implicit ec: ExecutionContext): Future[Option[CallCachingEntry]] = {
+    val action = for {
+      _ <- dataAccess.allowResultReuseForCallCachingEntryId(callCachingEntryId).update(false)
+      callCachingEntryOption <- dataAccess.callCachingEntriesForId(callCachingEntryId).result.headOption
+    } yield callCachingEntryOption
```

Comment:
The EJEA so it knows which call to update in the metadata to change the value of `allowResultReuse`